src/index.js

```diff
@@ -28,6 +27,9 @@ class SpatialReference {
     if (this.db) {
       getFromDB.call(this, wkid)
         .then(wkt => {
+          if (wkt) {
```

Comment:
With this conditional, line 34 will always be true and its conditional is superfluous.

Reply:
Yup, good call.